asapdiscovery-ml/asapdiscovery/ml/trainer.py

```diff
@@ -149,6 +168,17 @@ def output_dir_check(cls, p):
 
         return p
 
+    @validator("model_config")
+    def check_model_type_visnet_import(cls, v):
+        # VisNet requires PyG >=2.5.0. Currently only in PyG-nightly
+        # Refer MTENN issue #42
+        if not HAS_VISNET and isinstance(v, ViSNetModelConfig):
+            raise ImportError(
+                "Can't import ViSNetModelConfig without mtenn.conversion_utils.visnet."
+            )
+        else:
+            return v
```

Comment:
This validator also explicitly references `ViSNetModelConfig`

Reply:
@hmacdope any advice will be hugely appreciated!

Reply:
This shouldn't actually execute because HAS_VISNET is false right?

Reply:
the reference to ViSNetModelConfig will still throw a AttributeError